modules/clean.am

```diff
@@ -1,13 +1,6 @@
 #!/usr/bin/env bash
 
-# Check XDG variable
-if [ -n "$XDG_DATA_HOME" ]; then
-        DATADIR="$XDG_DATA_HOME"
-else
-        DATADIR="$HOME/.local/share"
-fi
-
-function _clean_launchers(){
+function _clean_launchers() {
```

Comment:
add comments for god's sake. i don't know why you are removing  DATADIR variable